src/display/annotation_layer.js

```diff
@@ -165,6 +165,10 @@ class AnnotationElement {
     let width = data.rect[2] - data.rect[0];
     let height = data.rect[3] - data.rect[1];
 
+    if (this.data.alternativeText) {
+      container.setAttribute("title", this.data.alternativeText);
```

Comment:
I think you can simplify this to `container.title = this.data.alternativeText;`. We should only use `setAttribute` for non-default attributes.